rio-code/src/main/java/frc/robot/Constants.java

```diff
@@ -96,6 +107,10 @@ public final class Manipulator {
 
         // Conversion Factor
         public final static double conversionFactor = 1;
+
+        public static final double detectionRange = 0.1;
+        public static final double sensorDebounce = 0.01;
+        public static final int sensorID = 18;
```

Comment:
If these are CAN Ids, can we specify that please? Also I'd like to discuss putting all the CAN IDs in one place in the constants file, as that makes it harder to accidentally overlap them.